### ISSUE TITLE
Add live template for config section

### ISSUE THREAD

**Issue by rdmueller** [asciidoctor/asciidoctor-intellij-plugin#358]
The pre-defined live templates are a great feature!
A template which shows examples for the various config attributes of AsciiDoc could be helpfull since it is hard to remember them all. Maybe we could cluster them:

```
ad-config-i18n
:caution-caption: Caution
:important-caption: Important
:note-caption: Note
:tip-caption: Tip
:warning-caption: Warning
:appendix-caption: Appendix
:example-caption: Example
:figure-caption: Figure
:table-caption: Table
:toc-title: Table of Contents
```
```
ad-config-toc
:toc-title: Table of Contents
:toc-placement: auto
:toclevels: 4
:sectnums:
:sectnumlevels: 2
:sectanchors:
:sectids:
:sectlinks:
```

```
ad-config-pdf
:pdf-page-size: A4
:pdf-stylesdir: ../pdfTheme

```

```
ad-config-render
:icons: font
:imagesdir: images
::doctype: book
```

I guess the change has to be applied to https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/master/src/main/resources/liveTemplates/AsciiDocLiveTemplates.xml

**Comment by mojavelinux** [comment]
`toc-placement` should not be used. The preferred is the consolidated `toc` attribute. For the default placement, you'd use:

```
:toc:
```

To make it a left sidebar (which only applies to the HTML), you'd use:

```
:toc: left
```

To use the macro, you'd use:

```
:toc: macro
```

**Comment by mojavelinux** [comment]
`pdf-stylesdir` is deprecated. The preferred attribute is `pdf-themesdir`.

**Comment by mojavelinux** [comment]
`sectids` is enabled by default, so there's no reason to specify it.

**Comment by mojavelinux** [comment]
You probably want to add `partnums` if you're adding `sectnums`.

**Comment by rdmueller** [comment]
@mojavelinux `partnums` seems only be described in the new docs, not in the current user manual

**Comment by ahus1** [comment]
I suppose ad-config-i18n and ad-config-toc would be the two most useful. I am not sure about the other two. 

I suggest adding line comments explaining each option to the user. The user can later remove the comments if they don't need them any more (there is Ctrl+Q to look up the docs for each attribute but not everyone knows about this).

**Comment by ahus1** [comment]
This is available as part of pre-release 0.30.17.
The new pre-release of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/master/CONTRIBUTING.adoc#eap).
